applications/MeshMovingApplication/custom_utilities/move_mesh_utilities.h

```diff
@@ -36,6 +36,13 @@ void MoveMesh(const ModelPart::NodesContainerType &rNodes);
 ModelPart* GenerateMeshPart(ModelPart &rModelPart,
                                     const std::string &rElementName);
 
+void SuperImposeVariables(ModelPart &rModelPart, const Variable< array_1d<double, 3> >& rVariable,
+                                                 const Variable< array_1d<double, 3> >& rVariableToSuperImpose);
+
+void SuperImposeMeshDisplacement(ModelPart &rModelPart, const Variable< array_1d<double, 3> >& rVariableToSuperImpose);
+
+void SuperImposeMeshVelocity(ModelPart &rModelPart, const Variable< array_1d<double, 3> >& rVariableToSuperImpose);
```

Comment:
are those necessary?
You could pass directly the first variable too

Reply:
strictly speaking not required but I liked this way because its more readable when using those functions.